Makefile

```diff
@@ -610,6 +627,59 @@ DO_PATCH    = cd $(BUILD_PATCH)/$(1); \
 		fi; \
 	done
 
+CONFIGURE_MAKE_INSTALL = cd $(BUILD_WORK)/$@; \
+	if ! [ $(1) ]; then \
+		./configure -C $(DEFAULT_CONFIGURE_FLAGS); \
+	else ./configure -C $(1); \
```

Comment:
I think it'd be cleaner to include DEFAULT_CONFIGURE_FLAGS regardless of whether extra autoconf flags are passed, since they're still getting included in makefiles where extra flags are being passed. Same for all the other install functions

Reply:
*custom configure scripts* sigh